src/MetadataUtility/Commands/EmuCommand.cs

```diff
@@ -21,6 +21,8 @@ public EmuCommand()
 
             this.AddGlobalOption(FormatOption);
 
+            this.AddGlobalOption(OutOption);
+
```

Comment:
I don't think we have support for changing the output do we?

In which case exposing this options is a breaking change?